docs/guides/production.md

```diff
@@ -0,0 +1,296 @@
+directory and you're all set.
+
+If you have additional, referenced assets that you want to be served by your fronting server, you can add an addition
+`location {}` block to point the server to that directory as well.
+
+All of this relies on proper configuration of `react-server` as mentioned in the example `.reactserverrc` file at the
+beginning of this guide.  Because `react-server` loads your compiled assets for you, you need to tell it where to find the assets.
+The `jsUrl` setting at the beginning of this guide does just that.  If you use `jsUrl: /assets/`, then `react-server` will
+try to load all of your assets using the relative URL `/assets`.
+
+## Served through a CDN
+Similar to a fronting server, the key here is to move all of the generated and referenced assets over to a CDN.  Once that
+is done, you need to modify `.reactserverrc` to set `jsUrl: http://mycdn.com/assets/`.  Note that in this case, the URL
+is absolute instead of relative.
+
+# Process & Cluster Managers
+Big Flashing Sign: Unhandled Javascript exceptions cause the node.js process to fail!  *Do not run `react-server` in production
+without some sort of process/cluster manager if your application may throw an unhandled exception or the world will end!*
+
+Read this first: [http://expressjs.com/en/advanced/pm.html](http://expressjs.com/en/advanced/pm.html)
+
+## `pm2`
+If using [pm2](http://pm2.keymetrics.io) to start `react-server`, add a file called `pm2.yml` to the top level directory
+of your application.  Also, you might want to have the `pm2` module installed globally by running the command `npm install -g pm2`.
+
+A sample `pm2.yaml` file is included below:
+
+```yaml
+apps:
+  - script : node_modules/react-server-cli/target/cli.js
+    args   : 'start'
+    name   : 'react-server'
+    watch  : true
+    ignore_watch: '__clientTemp'
+    env    :
+      NODE_ENV: development
+      REACT_SERVER_CONFIGS: _configs/development/
+    env_beta:
+      NODE_ENV: beta
+      REACT_SERVER_CONFIGS: _configs/beta/
+    env_production:
+      NODE_ENV: production
+      REACT_SERVER_CONFIGS: _configs/production/
+```
+
+This example follows the same setup as other parts of this guide by setting and passing the appropriate `NODE_ENV` and
+`REACT_SERVER_CONFIGS` variables to `react-server`.  Once the module is installed and the `pm2.yml` is in place, you can
+start the process using pm2 by running `pm2 start pm2.yml`.
+
+If you want, you can update the scripts section of `package.json` to interact with pm2 as well:
+
+```json
+{
+  "scripts": {
+    "start": "pm2 start pm2.yml --env development",
+    "start-beta": "pm2 start pm2.yml --env beta",
+    "start-prod": "pm2 start pm2.yml --env production"
+  }
+}
+```
+
+Now you can interact directly with the process using `pm2` commands or use `npm start`, `npm run start-beta`, or
+`npm run start-prod` to start the processes.
+
+## `recluster`
```

Comment:
oooo I'd never heard of recluster before, but it seems like it fits in to the ecosystem really well, and fills a pretty big gap.  Do you think we should ship with a recluster config out of the box?  Or add a `react-server-recluster` module to the monorepo?